### DISCUSSION TITLE
Test HIDAPI `set_nonblocking`

### DISCUSSION THREAD

**Discussion by JakubAndrysek** [JakubAndrysek/PySpaceMouse#14] [Ideas]
Look at [set_nonblocking](https://github.com/bglopez/python-easyhid/blob/master/easyhid/easyhid.py#LL202C17-L202C17) and test it wit the library.

**Comment by PeterMitrano** [comment]
I have tested this, and it works. See this branch for some usage: https://github.com/JakubAndrysek/PySpaceMouse/tree/read-latest
